### PR TITLE
starting passport facebook auth

### PR DESCRIPTION
**Pull request by sodapopinsky**

### CODE REVIEW

File: views/teams.hbs [hunk 1]
```diff
@@ -1,7 +1,11 @@
   <body>
 
+
     <div id="team-container">
-      <div id="team-select">What's your team?</div>
+	{{#loggedIn}}
```
Comment (blakeparkinson) [inline views/teams.hbs]: this isn't the right syntax for handlebars if clauses. should be ```{{#if loggedIn}}```

File: views/teams.hbs [hunk 2]
```diff
@@ -1,7 +1,11 @@
...
+   	 Welcome, {{ user.displayName }}
+	{{/loggedIn}}
```
Comment (blakeparkinson) [inline views/teams.hbs]: ```{/if}```

File: routes/teams.js [hunk 3]
```diff
@@ -25,15 +25,19 @@ var http = require("http"),
 
 
 
-router.get('/', function(res, res) {
-      res.render('teams', {
-      });
+router.get('/', function(req, res) {
+    var isLoggedIn = req.user ? true : false;
+  
+      res.render('teams', {user:req.user,loggedIn : isLoggedIn});
```
Comment (blakeparkinson) [inline routes/teams.js]: why pass 2 variables here when all you really need is req.user?

File: app.js [hunk 4]
```diff
@@ -24,13 +26,20 @@ app.use(logger('dev'));
 app.use(bodyParser.json());
 app.use(bodyParser.urlencoded({ extended: false }));
 app.use(cookieParser());
+
 app.use(express.static(path.join(__dirname, 'public')));
+app.use(session({ secret: '1e8804554c1f41249ae1b522b23f7937' }));
```
Comment (blakeparkinson) [inline app.js]: I dont think it's been merged yet, but let's put these in env.json when ready

File: routes/auth.js [hunk 5]
```diff
@@ -4,6 +4,16 @@ var http = require("http");
...
+  done(null, user);
+});
+
+passport.deserializeUser(function(obj, done) {
+  done(null, obj);
+});
+
+
 passport.use(new FacebookStrategy({
     clientID: '1600051886893474',
```
Comment (blakeparkinson) [inline routes/auth.js]: let's add these to env.json when ready. Then when can call ```config.clientID```

**Comment by blakeparkinson** [comment]
A couple things:

1. merge conflicts.
2. Why is the mysql module being included? We don't wanna use mysql

File: routes/teams.js (hunk 3 above)
Comment (sodapopinsky) [inline routes/teams.js]: Didn't know how do handlebars #if statement in teams.hbs without passing a boolean

File: views/teams.hbs [hunk 6]
```diff
@@ -1,6 +1,7 @@
   <body>
+
     <div id="team-container">
-      <div id="team-select">What's your team?</div>
+      <div id="team-select">What's your team? </div>
```
Comment (blakeparkinson) [inline views/teams.hbs]: :-1:

**Comment by blakeparkinson** [comment]
good job NTGD :camel:

**Pull request merged**